### DISCUSSION TITLE
codemaid for vscode

### DISCUSSION THREAD

**Discussion by AndersonLorena** [codecadwallader/codemaid#775] [Ideas]
Hi everyone, I'm not sure if this discussion happened before, however, I would like so much to start a thread about the viability to implement a variant codemaid for vscode. Thanks.

**Comment by Wastus** [comment]
There is an issue for this here: https://github.com/codecadwallader/codemaid/issues/273

**Reply by AndersonLorena** [reply]
Hi @Wastus, I will check, thanks =)

**Comment by codecadwallader** [comment]
Thanks for asking the question @AndersonLorena .  As @Wastus mentioned there is an existing issue which has a lot of background on the complexities and other thoughts.  Please be sure to check that out. :)

**Reply by AndersonLorena** [reply]
Hi @codecadwallader, I just followed the new link. thanks =)
